### PR TITLE
Add change of storage feature

### PR DESCRIPTION
**Pull request by advaypal**
Yet to write some more tests and implement rollback. Please review the rest

### CODE REVIEW

**Comment by Skaty** [comment]
Why the coverage decrease?

File: src/test/java/seedu/task/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -203,7 +204,16 @@ public void execute_addDuplicate_notAllowed() throws Exception {
 
     }
 
-
+    @Test 
+    public void execute_change_path_successful() {
```
Comment (Skaty) [inline src/test/java/seedu/task/logic/LogicManagerTest.java]: Why empty test?

**Review by Skaty — DISMISSED** [review]
Other than that, all good.

File: src/test/java/seedu/task/logic/LogicManagerTest.java (hunk 1 above)
Comment (advaypal) [reply src/test/java/seedu/task/logic/LogicManagerTest.java]: Yup adding tests. Hence coverage decrease

**Pull request merged**